CPP17.md

```diff
@@ -266,14 +266,20 @@ byte e = byte{256}; // ERROR
 
 ### fallthrough, nodiscard, maybe_unused attributes
 C++17 introduces three new attributes: `[[fallthrough]]`, `[[nodiscard]]` and `[[maybe_unused]]`.
-* `[[fallthrough]]` indicates to the compiler that falling through in a switch statement is intended behavior.
+* `[[fallthrough]]` indicates to the compiler that falling through in a switch statement is intended behavior. This attribure may only be used in a switch statement and next statement to be executed must be a case or default label. 
```

Comment:
Looks like there's a spelling mistake with "attribure". I also made the second part of the sentence more concise; can you change it to:

This attribute may only be used in a switch statement, and must be placed before the next case/default label.